requirements.txt

```diff
@@ -3,5 +3,5 @@ PyYAML==5.1
 confluent-kafka==0.11.6
 datadog==0.21.0
 jsonschema==3.0.1
-psycopg2-binary==2.7.7
+psycopg2-binary==2.9.3
```

Comment:
intentional?

Reply:
yeah the older version does not build on arm64